bin/main.ml

```diff
@@ -869,6 +870,32 @@ let subst =
          )
   , Term.info "subst" ~doc ~man)
 
+let utop =
+  let doc = "Load library in utop" in
+  let man = [ (* TODO *) ] in
+  let go common dir args =
+    let utop_target = dir |> Path.of_string |> Utop.target |> Path.to_string in
+    set_common common ~targets:[utop_target];
+    (* We must wait for other exit hooks to finish before forking. This is
```

Comment:
Indeed. There are other `at_exit` handlers but I think they don't need to be run at this point. Maybe we should have a better API for these at some point.